.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,14 +10,36 @@
 
 
 ### Issues Solved
-<!--- If this Pull Request solves any issues, please list them here  -->
-#
+<!--- If this Pull Request solves any issues, list them here, and mark the ticket "In Review" in the pipeline project -->
+Closes #
 
 ### Checklist
 <!--- Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.  -->
+-   I targeted one of these branches:
+    - [ ] dev/7.6.x (under development): features, bugfixes not covered below
+    - [ ] dev/7.5.x (main support): regressions, crashing bugs, security issues, major bugs in new features
+    - [ ] dev/6.2.x (extended support): major security issues, data loss issues
+-   [ ] I added a changelog in arches/releases
+-   [ ] I submitted a PR to arches-docs (if appropriate)
 -   [ ] Unit tests pass locally with my changes
--   [ ] I have added tests that prove my fix is effective or that my feature works
--   [ ] I have added necessary documentation (if appropriate)
+-   [ ] I added tests that prove my fix is effective or that my feature works
+-   [ ] My test fails on the target branch
+
+#### Accessibility Checklist
+<!-- Place a mark in the appropriate column for each topic -->
+[Developer Guide](https://arches.readthedocs.io/en/stable/developing/advanced/accessibility/)
+
+| Topic            | Unchanged | Retested |
```

Comment:
Instead of `Unchanged` maybe this column could be `Changed`? That way most PRs won't have to modify this table - only those PRs that actually might impact accessibility.